setup.py

```diff
@@ -2,11 +2,11 @@
 
 
 def version():
-    with open('v3iofs/__init__.py') as fp:
+    with open("v3iofs/__init__.py") as fp:
```

Comment:
Please refrain from such changes (`'` to `"`), it's just white noise. Try to have a little as possible in the PR